### PR TITLE
Es 1 5

### PR DESCRIPTION
**Pull request by EgozyN**

### CODE REVIEW

**Comment by EgozyN** [comment]
@synhershko

**Comment by synhershko** [comment]
can you rebase so it can be merged please?

**Comment by EgozyN** [comment]
Done

File: src/main/java/com/code972/elasticsearch/plugins/DictReceiver.java [hunk 1]
```diff
@@ -40,43 +38,19 @@ public static boolean setHebmorphDictionary(String dictheb_path) {
...
-
-    private static DictHebMorph setDictionary() {
+    private static DictHebMorph setDefaultDictionary() {
         File file;
-        for (String path : gZipFilePaths) {
+        for (String path : filePaths) {
             file = new File(path);
             if (file.exists()) {
                 try {
+<<<<<<< HEAD
```
Comment (synhershko) [inline src/main/java/com/code972/elasticsearch/plugins/DictReceiver.java]: faulty merge

File: README.md [hunk 2]
```diff
@@ -14,6 +16,19 @@ Powered by HebMorph (https://github.com/synhershko/HebMorph) and licensed under
 
 ~/elasticsearch-1.3.2$ bin/plugin --install analysis-hebrew --url http://dl.bintray.com/synhershko/HebMorph/elasticsearch-analysis-hebrew-1.5.zip
 
+~/elasticsearch-1.4.4$ bin/plugin --install analysis-hebrew --url http://dl.bintray.com/synhershko/HebMorph/elasticsearch-analysis-hebrew-1.6.zip
+(Coming soon)
+
+You will need hebrew dictionary files. The open-sourced hspell files can be downloaded here: https://github.com/synhershko/HebMorph/tree/master/hspell-data-files. Next, Elasticsearch needs to be configured to load the dictionary from that folder: this is done by adding the following line to elasticsearch.yml file:
+```
+    hebrew.dict.path: /PATH/TO/HSPELL/FOLDER
+```
```
Comment (synhershko) [inline README.md]: this is only true for the new ES plugin version

Comment (EgozyN) [inline README.md]: Do we want explanation on how to install older versions?

Comment (synhershko) [inline README.md]: You can move those to the bottom of the page, under "Older Versions"

**Pull request merged**